01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,22 @@
+# 初期値を0にする
+x=0
```

Comment:
後々rubocopを導入すれば機械的に指摘してもらえることだとは思いますが、一般的に演算子（`=`とか`<`とか`+`とか）の前後には半角スペースを入れた方が良いです。
```suggestion
x = 0
```

Reply:
演算子の前後に半角スペースを入れました。

Reply:
```suggestion
x = 0
```